### ISSUE TITLE
Is Horovod runs on Synchronize Batch Norm?

### ISSUE THREAD

**Issue by justanotherYO** [horovod/horovod#1275]
**Environment:**
1. Framework: Tensorflow
2. Framework version: 1.8
3. Horovod version: 0.15.1
4. MPI version:
5. CUDA version:9.0
6. NCCL version:
7. Python version:3.6
8. OS and version:
9. GCC version:

**Checklist:**
1. Did you search issues to find if somebody asked this question before? YES
2. If your question is about hang, did you read [this doc](https://github.com/horovod/horovod/blob/master/docs/running.rst)?
3. If your question is about docker, did you read [this doc](https://github.com/horovod/horovod/blob/master/docs/docker.rst)?
4. Did you check if you question is answered in the [troubleshooting guide](https://github.com/horovod/horovod/blob/master/docs/troubleshooting.rst)?

**Your question:**

I am wondering that could Horovod do Synchronize Batch Norm when training on Multi-GPUS? If so, how? do I have to customize the TensorFlow BN layers or Horovod can do it automatically for me?

I am doing a segmentation job with High volumn images, and my batch size can poorly be 1 or 2. So using multi-GPUs with syncBN is important for me. 

Thank you very much for any informations!

**Comment by andvidal** [comment]
I have used [tensorpack](https://github.com/tensorpack/tensorpack) for this, where local mean and variances and aggregated across multiple gpus, so there are global statistics rather than local ones, as is the case for horovod
